plugins/modules/web_infrastructure/nomad_job.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, FERREIRA Christophe <christophe.ferreira@cnaf.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: nomad_jobs
+author: "FERREIRA Christophe"
+version_added: "2.10"
+short_description: Launch an Nomad Job.
+description:
+    - Launch an Nomad job. See
+      U(https://www.nomadproject.io/api-docs/jobs/) for an overview.
+options:
+    host:
+      description:
+        - fqdn of nomad server.
+      required: True
+    secure:
+      description:
+        - ssl connection.
+    timeout:
+      description:
+        - timeout of request nomad.
+    verify:
+      description:
+        - skip validation cert.
+    cert:
+      description:
+        - path of certificate ssl.
+    key:
+      description:
+        - path of certificate key ssl.
+    namespace:
+      description:
+        - namespace for nomad.
+    token:
+      description:
+        - acl token for authentification.
+    name:
+      description:
+        - name of job for get|stop.
+    state:
+      description:
+        - type of request
+      choices: ["create", "stop", "get", "list"]
```

Comment:
`get` and `list` should not appear in a regular module. Create an `_info` module for that.

Also, better use state names such as `created`, `stopped`, `started`, ..., instead of action names.